java/server/src/org/openqa/selenium/grid/console/package-lock.json

```diff
@@ -0,0 +1,15811 @@
+{
```

Comment:
I think this file is not required and can be removed.

Reply:
https://stackoverflow.com/a/44210813/8608146 it should be checked in source control?

Reply:
It's required.